apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.test.js

```diff
@@ -56,6 +56,10 @@ function generateQRCode (qrCodeData = {}, { version = '0001', encodingTag = '2'
         ...qrCodeData,
     }
 
+    // Should work case-insensitively
+    qrCodeObj.persAcc = qrCodeObj.PersAcc
+    delete qrCodeObj['PersAcc']
```

Comment:
looks like some strange hacks

Reply:
Why? I'd like to keep value if passed via PersAcc. Also I'd like to prevent appearing of `PersAcc=undefined` within stringified qr-code.